pom.xml

```diff
@@ -251,10 +262,10 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
-                <version>3.3</version>
+                <version>3.8.1</version>
                 <configuration>
-                    <source>1.8</source>
-                    <target>1.8</target>
+                    <source>1.9</source>
```

Comment:
Do we have to drop support for Java 8? We can have Travis build on 8 and 9 to test that both work